.travis.yml

```
@@ -42,10 +43,19 @@ jobs:
         - sbt ++$TRAVIS_SCALA_VERSION idlgen-core/test
         - sbt ++$TRAVIS_SCALA_VERSION idlgen-sbt/test
         - sbt ++$TRAVIS_SCALA_VERSION publishLocal idlgen-sbt/publishLocal idlgen-sbt/scripted
+    - stage: deployPlugin
+      scala: 2.12.8
+      if: branch = master AND type != pull_request AND env(SKIP_SBT_DEPLOY) = false
+      script:
+        - if grep -q "SNAPSHOT" version.sbt; then
+            sbt idlgen-sbt/publish;
+          else
+            sbt ";project idlgen-sbt;release";
```

```suggestion
            sbt idlgen-sbt/release;
```

It's minor suggestion, feel free to discard it